Cython/Build/__init__.py

```diff
@@ -3,8 +3,17 @@
 import sys
 if sys.version_info < (3, 7):
     from .Distutils import build_ext
-del sys
 
+if sys.version_info > (3, 11):
```

Comment:
I don't think this version check is correct.

Reply:
Hm I am not sure why? Could you be more specific?

Reply:
```
>>> (3, 11, 1) > (3, 11)
True
```

Reply:
Now I see. Thanks!